e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/security/notrecognized/SecurityNotRecognizedEMTest.kt

```diff
@@ -43,10 +43,10 @@ class SecurityNotRecognizedEMTest : SpringTestBase(){
             assertHasAtLeastOne(solution, HttpVerb.GET, 403, "/api/resources/{id}", null)
             assertHasAtLeastOne(solution, HttpVerb.POST, 401, "/api/resources/", null)
 
+            val faults = DetectedFaultUtils.getDetectedFaultCategories(solution).filter { it == DefinedFaultCategory.SECURITY_NOT_RECOGNIZED_AUTHENTICATED }
```

Comment:
hmmmm... i need to look into this. eg was a SECURITY_FORGOTTEN_AUTHENTICATION found here?

Reply:
@omursahin  I looked at the code, and i don't think there is a `SECURITY_FORGOTTEN_AUTHENTICATION` here. the fact that is found might a bug in the oracle check. can you double-check it?